### PR TITLE
sorting implemented

### PR DESCRIPTION
**Pull request by Szpytma**

### CODE REVIEW

File: src/components/Sorting.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          <Button>Sort By date</Button>
+        </Link>
+        <Link to={`?order=${isAsc ? "asc" : "desc"}&sort_by=comment_count`}>
+          <Button>Sort By comment count</Button>
+        </Link>
+
+        <Form.Check
+          type="checkbox"
+          id="asc"
+          checked={isAsc}
```
Comment (roisinoreilly) [inline src/components/Sorting.jsx:50]: it's really not clear to the user that checking this box will flip from ascending to descending

File: src/components/Sorting.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  }
+  return (
+    <div>
+      <div>
+        <p>Order by:</p>
+        <Link to={`?order=${isAsc ? "asc" : "desc"}&sort_by=votes`}>
+          <Button>Sort By Votes count</Button>
+        </Link>
+        <Link to={`?order=${isAsc ? "asc" : "desc"}&sort_by=created_at`}>
+          <Button>Sort By date</Button>
```
Comment (roisinoreilly) [inline src/components/Sorting.jsx:36]: dropdowns would be much more user friendly than buttons here, especially for the sorting

**Review by roisinoreilly — CHANGES_REQUESTED** [review]
this is looking really good! Functionality wise everything works great, I've left a few comments about the user experience I'd like you to have a look at, after you've sorted that send me a DM or put in another PR :)

**Review by lewispricey — CHANGES_REQUESTED** [review]
Great work on this ticket so far, I really like the catogory dropdown and your buttons to change the sort order.

Currently to change between asc and desc I have to tick/untick the box and then press one of the sort buttons again. Whilst this works it doesnt make for the most straight forward UI.

It would be nice to perhaps have a dropdown for Asc/Desc and when that changes update the sort automaticaly without me having to click a button.

Overall great stuff the apps coming on really well, please could you implement the dropdown or change your toggle so this updates the query without having to do anything extra and raise another PR.

**Review by lewispricey — APPROVED** [review]
Awesome! Thanks for implementing those changes 👍 

Good to merge when you're ready 🚀

**Pull request merged**